.github/workflows/publish.yml

```diff
@@ -19,15 +19,15 @@ jobs:
       - name: Install node
         uses: actions/setup-node@v2
         with:
-          node-version: "16.13.1"
+          node-version: "18.12.1"
```

Comment:
So far we've (at least mostly?) been less strict here, locking the major version only (`^18.12.1`)

Reply:
This is for the Github action. We have to specify the exact version here (and always have). In the `package.json` we (mostly) use `^`.

Reply:
I know what the setup is for of course, I just mixed where we locked them and how. Ok 👍